src/awscdk-construct.ts

```diff
@@ -225,10 +250,63 @@ export class AwsCdkConstructLibrary extends ConstructLibrary {
   private formatModuleSpec(module: string): string {
     return `${module}@${this.cdkVersion}`;
   }
+
+  /**
+   * Enables CDK apps to consume construct library lambdas.
+   *
+   * @param bundleLambdaOptions Bundle lambda options
+   * @private
+   */
+  private bundleLambdas(bundleLambdaOptions?: BundleLambdaOptions) {
+    this.logger.debug('running bundleLambdas');
+    const lambdaSuffix = (bundleLambdaOptions?.suffix ?? '.lambda');
+    const generateConstruct = bundleLambdaOptions?.generateConstruct ?? true;
+    const handlerFilePattern = `${this.srcdir}/**/*${lambdaSuffix}\\.@(ts|js)`;
+
+    this.addDevDeps('aws-sdk'); // common lambda runtime dependency
+    this.addDevDeps('esbuild');
+
+    this.eslint?.addOverride({
+      rules: {
+        'import/no-extraneous-dependencies': 'off',
+      },
+      files: [handlerFilePattern],
+    });
+
+    // create bundle compile task for each lambda handler, generate convenience construct
+    glob.sync(handlerFilePattern).forEach(file => {
+      this.logger.debug(`found lambda handler ${file}`);
+      const parsedPath = path.parse(file);
+      const bundleDirName = `${parsedPath.name}.bundle`;
+      const bundleDir = path.join(parsedPath.dir, `${parsedPath.name}.bundle`);
+      const compiledHandlerPath = bundleDir.replace(this.srcdir, this.libdir);
+      const constructFilePath = path.join(parsedPath.dir, parsedPath.name.replace(lambdaSuffix, '') + '.ts');
+
+      if (generateConstruct) {
+        new NodeLambdaConstructSourceCode(this, constructFilePath, bundleDirName);
+      }
+      const bundleLambdaTask = this.addTask(`bundleLambda:${file}`, {
```

Comment:
I think in this context, "bundle:" is sufficient as a prefix.

```suggestion
      const bundleLambdaTask = this.addTask(`bundle:${file}`, {
```